buildkitd/settings.go

```diff
@@ -1,57 +1,47 @@
 package buildkitd
 
 import (
-	"crypto/sha256"
-	"encoding/base64"
-	"encoding/json"
+	"strconv"
+	"strings"
+	"time"
 
+	"github.com/mitchellh/hashstructure/v2"
 	"github.com/pkg/errors"
-	"golang.org/x/crypto/bcrypt"
 )
 
 // Settings represents the buildkitd settings used to start up the daemon with.
 type Settings struct {
-	CacheSizeMb      int      `json:"cacheSizeMb"`
-	GitURLInsteadOf  string   `json:"gitUrlInsteadOf"`
-	Debug            bool     `json:"debug"`
-	DebuggerPort     int      `json:"debuggerPort"`
-	AdditionalArgs   []string `json:"additionalArgs"`
-	AdditionalConfig string   `json:"additionalConfig"`
-	CniMtu           uint16   `json:"cnuMtu"`
+	CacheSizeMb      int
+	GitURLInsteadOf  string
+	Debug            bool
+	DebuggerPort     int
+	AdditionalArgs   []string
+	AdditionalConfig string
+	CniMtu           uint16
+	Timeout          time.Duration `hash:"ignore"`
```

Comment:
I like the `hash:"ignore"`

Reply:
Yes, me too. It makes life better. Fun fact: one of our deps already used this, so no real harm!